docs/ai-coder/tasks.md

```diff
@@ -119,6 +119,237 @@ Coder can automatically generate a name your tasks if you set the `ANTHROPIC_API
+
+    - List all tasks you can view.:
+
+        $ coder exp task list --all
+
+    - List all your running tasks.:
+
+        $ coder exp task list --status running
+
+    - As above, but only show IDs.:
+
+        $ coder exp task list --status running --quiet
+
+OPTIONS:
+  -a, --all bool (default: false)
+          List tasks for all users you can view.
+
+  -c, --column [id|organization id|owner id|owner name|name|template id|template name|template display name|template icon|workspace id|workspace agent id|workspace agent lifecycle|workspace agent health|initial prompt|status|state|message|created at|updated at|state changed] (default: name,status,state,state changed,message)
+          Columns to display in table output.
+
+  -o, --output table|json (default: table)
+          Output format.
+
+  -q, --quiet bool (default: false)
+          Only display task IDs.
+
+      --status string
+          Filter by task status (e.g. running, failed, etc).
+
+      --user string
+          List tasks for the specified user (username, "me").
+```
+
+### Viewing Task Logs
+
+```console
+USAGE:
+  coder exp task logs [flags] <task>
+
+  Show a task's logs
+
+    - Show logs for a given task.:
+
+        $ coder exp task logs task1
+
+OPTIONS:
+  -c, --column [id|content|type|time] (default: type,content)
+          Columns to display in table output.
+
+  -o, --output table|json (default: table)
+          Output format.
+```
+
+### Send input to a task
+
+```console
+USAGE:
+  coder exp task send [flags] <task> [<input> | --stdin]
+
+  Send input to a task
+
+    - Send direct input to a task.:
+
+        $ coder exp task send task1 "Please also add unit tests"
+
+    - Send input from stdin to a task.:
+
+        $ echo "Please also add unit tests" | coder exp task send task1 --stdin
+
+OPTIONS:
+      --stdin bool
+          Reads the input from stdin.
+```
+
+### Viewing Task Status
+
+```console
+USAGE:
+  coder exp task status [flags]
+
+  Show the status of a task.
+
+  Aliases: stat
+
+    - Show the status of a given task.:
+
+        $ coder exp task status task1
+
+    - Watch the status of a given task until it completes (idle or stopped).:
+
+        $ coder exp task status task1 --watch
+
+OPTIONS:
+  -c, --column [state changed|status|healthy|state|message] (default: state changed,status,healthy,state,message)
+          Columns to display in table output.
+
+  -o, --output table|json (default: table)
+          Output format.
+
+      --watch bool (default: false)
+          Watch the task status output. This will stream updates to the terminal until the underlying workspace is stopped.
+```
+
+> **Note**: The `--watch` flag will automatically exit when the task reaches a terminal state. Watch mode ends when:
+>
+> - The workspace is stopped
+> - The workspace agent becomes unhealthy or is shutting down
+> - The task completes (reaches a non-working state like completed, failed, or canceled)
```

Comment:
I think this is all good documentation to have. Having it on the main Tasks documentation page, which is already pretty long, seems like too much. 

Can you alternatively 
1. Make a new sub page under `ai-coder/tasks/cli` and add this documentation to it
2.  Then leave a reference on this main page and the [Tasks Core Principles](https://coder.com/docs/ai-coder/tasks-core-principles) page that we have a CLI and to go read more at the new sub page

We can keep adding/sprucing up the CLI-specific page as development progresses!